.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+
+<!-- 
+
+Sorry to hear you're having trouble! If you have a question, please ask in 
+[SO or gitter](https://github.com/callemall/material-ui/blob/master/CONTRIBUTING.md#asking-questions)
```

Comment:
Since this won't be rendered as markdown, maybe we could just put two direct links to those areas links (possibly shortened. What do you think?